first commit

PallaviRIyer opened the pull request.

PallaviRIyer committed:
first commit

PallaviRIyer committed:
Create README.md

PallaviRIyer committed:
second commit

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/applibgroup/smart_watch_seven?pullRequest=1) <br/>All committers have signed the CLA.

PallaviRIyer committed:
third commit

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/applibgroup/smart_watch_seven?pullRequest=1) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/applibgroup/smart_watch_seven?pullRequest=1) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/applibgroup/smart_watch_seven?pullRequest=1) it.</sub>

PallaviRIyer committed:
mysteps

PatExplores commented:
1. Remove Start step count button, embed Step count by default.

PatExplores reviewed (changes requested):
1. remove .idea folder

PatExplores reviewed (commented):
1. change package name like com.example.smartwatchseven
2. Create "screenshot" folder add snippets(GIF) of app screen
3. Update Readme.md describing all the features of application
4. Remove entry/src/main/js/default/common/MyApplication.zip
6. check can we remove wallpaper black image and set background color to black.
7. Remove start step count button, start on app launch and stop on exit
8. css in place of pixel values, choose border alignments and margins,. Becuase pixels might not works well with bigger or smaller screens.
9. index.js -> btnText, Give meaningful name
10. index.js -> Use array to fetch Month and weekname
11. fetch weather info using current location
12. calory progress not implemented,

Ajith6901 committed:
1.Package name changed
2.deleted .zip file from common folder
3.removed Wallpaper.png and added background color

PallaviRIyer committed:
Merge pull request #1 from Ajith6901/main

First try to run the project , then only merge !

PallaviRIyer committed:
Used array to fetch month and weekname

Ajith6901 committed:
"readme"

Ajith6901 committed:
Merge branch 'PallaviRIyer:main' into main

Ajith6901 committed:
"Resolved some comments"

PallaviRIyer committed:
Merge pull request #2 from Ajith6901/main

Please merge my PR
